StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/STPAnalyticsClient+PaymentSheet.swift

```diff
@@ -86,7 +86,7 @@ extension PaymentSheet.Appearance.EmbeddedPaymentElement {
 
     var analyticPayload: [String: Any] {
         var payload = [String: Any]()
-        payload["style"] = style != PaymentSheet.Appearance.EmbeddedPaymentElement.default.style
+        payload["style"] = row.style != PaymentSheet.Appearance.EmbeddedPaymentElement.default.row.style
         payload["row"] = row != PaymentSheet.Appearance.EmbeddedPaymentElement.default.row
```

Comment:
While we still have the chance (pre beta/GA), seems like we should update the key for this to `row_style`?

Reply:
Hmm at some point in time, we should update our `PaymentSheetAnalyticsHelperTest` to include unit tests for this.